src/components/tier.jsx

```diff
@@ -42,7 +41,6 @@ class Tier extends React.Component {
             item2: data.item2,
```

Comment:
I'm probably not seeing the full picture here inside the for loop so correct me if I'm wrong but wouldn't every loop through data.length just keep setting the state of item1 & item2 to data.item1 & data.item2? If you're using for testing purposes ignore this comment lol

Reply:
Yeah, my next PR fixed this, I believe. Good catch :)